inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -428,3 +428,10 @@ def stop_matched_holdingpen_wfs(obj, eng):
         # stop this holdingpen workflow by replacing its steps with a stop step
         holdingpen_wf_eng.callbacks.replace(stopping_steps)
         holdingpen_wf_eng.process([holdingpen_wf])
+
+
+@with_debug_logging
+def has_more_than_one_exact_match(obj, eng):
+    """Does the record have more than one exact match."""
+    exact_matches = obj.extra_data['matches']['exact']
+    return len(set(exact_matches)) > 1
```

Comment:
A record can exact match with a `DOI` and an `arxiv_eprint`. This means we can have twice the same id in 'exact_matches'. So, we should check that, in case the `len > 1`, the ids are different.